extensions/wikia/AnalyticsEngine/AnalyticsProviderAmazonMatch.php

```diff
@@ -1,5 +1,6 @@
 <?php
 
+// TODO ADEN-5756 Remove AmazonMatch after A9 full rollout
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this "TODO" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)